### PR TITLE
Add Vault `/export` support 

### PR DESCRIPTION
**Pull request by kenany**

### CODE REVIEW

File: packages/pangea-node-sdk/tests/integration/vault.test.ts [hunk 1]
```diff
@@ -777,3 +783,16 @@ it("encrypt transform", async () => {
   expect(decrypted.result.id).toStrictEqual(key);
   expect(decrypted.result.plain_text).toStrictEqual(plainText);
 });
+
+it("export", async () => {
+  const key = await asymGenerateDefault(
```
[the remark this answers is not in the document]
Comment (kenany) [reply packages/pangea-node-sdk/tests/integration/vault.test.ts:789]: Done.

File: packages/pangea-node-sdk/src/types.ts [hunk 2]
```diff
@@ -1086,6 +1156,9 @@ export namespace Vault {
       rotation_frequency?: string;
       rotation_state?: ItemVersionState;
       expiration?: string;
+
+      /** Whether the key is exportable or not. */
+      exportable?: boolean;
```
[the remark this answers is not in the document]
Comment (kenany) [reply packages/pangea-node-sdk/src/types.ts:1167]: Done.

File: packages/pangea-node-sdk/src/types.ts [hunk 3]
```diff
@@ -1124,14 +1203,6 @@ export namespace Vault {
       secret: string;
     }
 
-    export interface GenerateRequest
```
Comment (pangea-andrest) [inline packages/pangea-node-sdk/src/types.ts:1127]: Why these were removed? Aren't they used?

**Review by pangea-andrest — DISMISSED** [review]

File: packages/pangea-node-sdk/src/types.ts (hunk 3 above)
Comment (kenany) [reply packages/pangea-node-sdk/src/types.ts:1127]: They are not used.

**Review by pangea-andrest — APPROVED** [review]

**Pull request merged**